lib/components/form/LinkInverters.tsx

```diff
@@ -55,17 +63,52 @@ const LinkInverters: FC<{ siteUUID: string }> = ({ siteUUID }) => {
         data, providing you with better forecasts and more information available
         within our app.
       </div>
-
       {showInfo && (
         <button
-          className="mt-[5px] block w-full self-center text-right text-[14px] text-ocf-yellow underline md:hidden"
+          className="mt-[5px] w-full self-center text-center text-[14px] text-ocf-yellow underline md:hidden"
           onClick={() => setShowInfo(false)}
         >
           Show less
         </button>
       )}
 
-      <a className="mt-auto self-center md:mt-5">
+      <div className="block self-center md:hidden">
+        <div className="mt-10 self-center text-ocf-gray-300">
+          Supported Inverters
+        </div>
+        <div className="mt-3 max-h-44 self-center overflow-y-scroll px-6">
+          <div className="flex flex-row">
+            <img src={ema.src} alt="ema"></img>
+            <div className="my-2 self-center text-ocf-gray-300">EMA</div>
+          </div>
+          <div className="flex flex-row">
+            <img src={enphase.src} alt="ema"></img>
+            <div className="my-2 self-center text-ocf-gray-300">Enphase</div>
+          </div>
+          <div className="flex flex-row">
+            <img src={fronius.src} alt="ema"></img>
+            <div className="my-2 self-center text-ocf-gray-300">Fronius</div>
+          </div>
+          <div className="flex flex-row">
+            <img src={goodwe.src} alt="ema"></img>
+            <div className="my-2 self-center text-ocf-gray-300">GoodWe</div>
+          </div>
+          <div className="flex flex-row">
+            <img src={growatt.src} alt="ema"></img>
+            <div className="my-2 self-center text-ocf-gray-300">Growatt</div>
+          </div>
+          <div className="flex flex-row">
+            <img src={solaredge.src} alt="ema"></img>
+            <div className="my-2 self-center text-ocf-gray-300">SolarEdge</div>
+          </div>
+          <div className="flex flex-row">
+            <img src={solis.src} alt="ema"></img>
+            <div className="my-2 self-center text-ocf-gray-300">Solis</div>
```

Comment:
We could maybe make this code simpler by using the same idea from the help pages—some object mapping brands to srcs?

Reply:
Yes, I was planning on doing that actually!